README.md

```diff
@@ -566,6 +566,31 @@ define(['doc'], function(doc) {
 });
 ```
 
+#### insertBefore
+`.insertBefore()`
+
+###### Description:
+Inserts an element before each matched element
+
+###### Sample:
+``` js
+define(['doc'], function(doc) {
+	doc('#some-element').insertBefore("a") //Inserts the element with id some-element before each <a> element
+});
+```
+#### insertAfter
+`.insertAfter()`
+
+###### Description:
+Inserts an element after each matched element
+
+###### Sample:
+``` js
+define(['doc'], function(doc) {
+	doc('#some-element').insertAfter("a") //Inserts the element with id some-element after each <a> element
```

Comment:
You should add the example where you pass a doc element instead of a string as well.

@mottam @erikatakahara